common/views.py

```diff
@@ -6,8 +6,10 @@
 from django.db.models import Case, When
 from django.core.cache import cache
 
-from common.selection import SimpleSelection, Selection, SelectionItem
 from common import definitions
+Alignment = getattr(__import__('common.alignment_' + settings.SITE_NAME, fromlist=['Alignment']), 'Alignment')
+
```

Comment:
This looks like an odd placement, but it makes sense given its use in 1382.

Reply:
Agreed, but just trying to be consistent 😁